src/main/java/com/gooddata/dataload/processes/ProcessService.java

```diff
@@ -95,6 +97,33 @@ public DataloadProcess createProcess(Project project, DataloadProcess process) {
     }
 
     /**
+     * Create new ruby process from ruby appstore.
+     * Processes can be deployed only from predefined repos, which are substituted by placeholders
+     *
+     * @param project project to which the process belongs
+     * @param process to create
+     * @param repoPlaceholderName name of repo placeholder, (e.g. PUBLIC_APPSTORE)
+     * @param commitIdentifier string in format commit/COMMIT_HASH or tag/TAG or branch/BRANCH
+     * @param pathInRepo string representing path in git repo to application (e.g. /test/rubyHello)
+     * @return created process
+     */
+    public FutureResult<DataloadProcess> createProcess(Project project, DataloadProcess process, String repoPlaceholderName, String commitIdentifier,
```

Comment:
do we really need a new method? isn't a app store process just another type of DataloadProcess?